app/assets/stylesheets/app/consent_form_preview.scss

```diff
@@ -0,0 +1,38 @@
+.consent-agreement {
```

Comment:
My standard way of going about this is to extract on the second or third use case such that the context that that second or third use case brings can be applied. It seems to me less possible to do it now because I'll be bringing out a crystal ball as to how it might be applied to a project I've not seen (also I don't feel so qualified to contribute to a house style without that having some origin in @grockett's designs).

That's [this school](http://wiki.c2.com/?PrematureGeneralization) of thought. To me generalisation is something that's done grudgingly, after you've seen the same thing multiple times and need a common pattern for it. Is CSS different? Are there any good articles that can help me understand why?